runbotics/runbotics-actions-windows/src/automation/office/excel.action-handler.ts

```diff
@@ -283,19 +248,16 @@ export default class ExcelActionHandler extends StatefulActionHandler {
     ): Promise<void> {
         try {
             const targetWorksheet = this.session.Worksheets(input?.worksheet ?? this.session.ActiveSheet.Name);
-            if (!Array.isArray(input.targetCells))
-                targetWorksheet
-                    .Range(input.targetCells)
-                    .Clear();
-            else for (const cellCoordinate of input.targetCells)
-                targetWorksheet
-                    .Range(cellCoordinate)
-                    .Clear();
+            if (!Array.isArray(input.targetCells)) targetWorksheet.Range(input.targetCells).Clear();
+            else for (const cellCoordinate of input.targetCells) targetWorksheet.Range(cellCoordinate).Clear();
         } catch (e) {
             throw new Error(ExcelErrorMessage.clearCellsIncorrectInput(e));
         }
     }
 
+    async worksheetExist(input: ExcelWorksheetExistInput): Promise<unknown> {
```

Comment:
worksheetExists (with s at the end, we check one worksheet), but I think I would name it doesWorksheetExist or isWroksheetPresent since the output is a boolean, but maybe others have better suggestion